src/components/Nav/Nav.js

```diff
@@ -0,0 +1,38 @@
+import { Container, Logo, Menu, Text } from "./Nav.styles";
+
+function Nav() {
+  return (
+    <Container>
+      <Logo>
+        <Text color="blue">Allcorrect</Text>
+        <Text>Game outsourcing studio</Text>
+      </Logo>
+      <Menu>
+        <Text as="a" href="#" rel="noreferrer noopener" className="normal">
```

Comment:
I think these navigation items should direct users to other parts of the page. Instead of using the Text component, it would be better to use Link from Next.js You can check out the [link](https://nextjs.org/docs/api-reference/next/link). I have used Link for the footer component.

Reply:
yea, you are right 
but now there is no page yet .. thats why i used anchor tags then will use Link after implementing pages